pkg/route/api/v1/types.go

```diff
@@ -43,6 +43,10 @@ type RouteSpec struct {
 	// be defaulted to Service.
 	To kapi.ObjectReference `json:"to"`
 
+	// AdditionalTos is an extension of the 'to' field. If more than one service needs to be
+	// pointed to, then use this field
+	AdditionalTos []kapi.ObjectReference `json:"additionalTos,omitempty"`
```

Comment:
Possible, although we don't have to use the same object.  And the
internal name is primarily for clarify on internal code, rather than
for sharing (probably no code would share the internal object between
deployments and routes).